examples/search-index/searchIndexFields.yaml

```diff
@@ -0,0 +1,39 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
[q] Does the manual QA docs in the repo also guide regression testing to use this example also?

Reply:
no, good catch. I will update the manual QA file in the follow-up PR for the CDK L1 constructor